src/Lecture1.hs

```diff
@@ -66,8 +63,8 @@ sumOfSquares x y = error "TODO!"
 🕯 HINT: use the @mod@ function
 
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = mod (abs n) 10
```

Comment:
Absolutely correct 👍🏻 
Alternatively, you can use the infix form of a function by putting it inside backticks :slightly_smiling_face: 
```suggestion
lastDigit n = abs n `mod` 10
```